_includes/feature-state-alpha.md

```diff
@@ -0,0 +1,13 @@
+{% assign dialog_title = "alpha" %}
+{% capture dialog_content %}
+This feature is currently in a `alpha` state, meaning:
+
+* The version names contain alpha (e.g. v1alpha1).
+* May be buggy. Enabling the feature may expose bugs. Disabled by default.
+* Support for feature may be dropped at any time without notice.
```

Comment:
I pulled the API versioning copy from https://kubernetes.io/docs/reference/api-overview/, which was left there from before.